CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ x.x.x Release notes (yyyy-MM-dd)
 
 * Deprecated `migrateRealm:` in favor of new `performMigrationForConfiguration:error:` method
   that follows Cocoa's' NSError conventions.
+* Attempting to add an object with no properties to a Realm now throws.
```

Comment:
"now throws rather than silently doing nothing." to make it clear that it didn't actually work before.